kd_sweep.sh

```
@@ -50,6 +87,17 @@ elif [[ "$MODE" == "coarse_k" ]]; then
     fi
   done
 
+elif [[ "$MODE" == "compare_methods_gsm8k" ]]; then
+  # Run all three methods at fixed k
+  for METHOD in top-k-tok; do
```

Shellcheck (0.11.0)

[warning] This loop will only ever run once. Bad quoting or missing glob/expansion?

(SC2043)